### PR TITLE
Pagination, switched to apollo hooks.

### PR DESCRIPTION
**Pull request by snacksncode**
**Changes:** 
- Added pagination
- Switched from `<Query>` to `useQuery()`
- Added markdown to blogEntry's subtitle
- Minor changes

### CODE REVIEW

File: components/Footer/Footer.jsx [hunk 1]
```diff
@@ -17,111 +21,106 @@ const articlesQuery = gql`
...
+            <br />
+            Telefon: <a href="tel:+48-338-159-045">(33) 815-90-45</a>
+          </div>
+
+          <iframe
+            src="https://www.facebook.com/plugins/page.php?href=https%3A%2F%2Fwww.facebook.com%2Fbielskiepogotowieratunkowe%2F&tabs=timeline&width=340&height=120&small_header=false&adapt_container_width=true&hide_cover=false&show_facepile=true&appId"
+            width="340"
+            title="facebook"
+            height="120"
+            style={{ border: 'none', overflow: 'hidden' }}
```
Comment (Darex1991) [inline components/Footer/Footer.jsx:66]: please move this to global styles

File: components/NavigationButtons/navigationButtonsStyles.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+@import '../../styles/variables';
+
+.pagination-buttons {
+  &__container {
```
Comment (Darex1991) [inline components/NavigationButtons/navigationButtonsStyles.scss:4]: please merge it with parent

File: components/SideBar/sideBarStyles.scss [hunk 3]
```diff
@@ -7,7 +7,7 @@
   width: 100%;
 }
 
-@media (min-width: $max-width-cap) {
+@media (min-width: 992px) {
```
Comment (Darex1991) [inline components/SideBar/sideBarStyles.scss:10]: please use variables

**Comment by Darex1991** [comment]
@snacksncode navigation is wrong. Again you didn't check mockups!
Screenshot 2020-06-29 at 10 25 58

**Comment by snacksncode** [comment]
Oh there is navigation there. Didn't look at it. Fixable

**Comment by snacksncode** [comment]
- added custom 404 page
- changed name from `NavigationButtons` to `PaginationButtons` to more clearly represent component
- pagination buttons styled according to mock-up
- minor changes in error handling

**Comment by Darex1991** [comment]
1) Screenshot 2020-06-29 at 23 08 45
2) focused btn shoudn't be blue
Screenshot 2020-06-29 at 23 07 56

**Pull request merged**